File: java/test/_polytrack7.java
import liba.polytrack7;

class _polytrack7 {
    public static void main(String[] args) {
        polytrack7 obj = new polytrack7(0, 1, 0, 1, 0, 1, 0, 1, 0, 1);
        double[] out = obj.out(0.5);
        System.out.print(obj.pos(0.5) + "," + obj.vel(0.5) + "," + obj.acc(0.5) + "," + obj.jer(0.5));
        System.out.println(" " + out[0] + "," + out[1] + "," + out[2] + "," + out[3]);
    }
}

File: quickjs/test/polytrack7.js
import * as a from "liba.so";
var polytrack7 = new a.polytrack7(0, 1, 0, 0)
polytrack7.gen(0, 10, 0, 10, 0, 0)
for (var i = 0; i < 10; i += 0.1) {
    console.log(i, polytrack7.pos(i), polytrack7.vel(i), polytrack7.acc(i), polytrack7.jer(0.5))
}
